reducers/plugins/index.ts

```diff
@@ -49,7 +55,16 @@ function sortComponents(a, b) {
     return 0;
 }
 
-function removePostPluginComponents(state, action) {
+type ComponentsState = {
```

Comment:
If you wanted to avoid redefining these, you could import `PluginsState` from `types/plugins` and type things like `PluginsState['components']` and `PluginsState['postTypes']`. Same with the various other reducers in here

Reply:
Nice one, I'm changing it.